### PR TITLE
Vet clinic database: database performance audit

### PR DESCRIPTION
**Pull request by RaphDasilva**
## We did the following.

- [ ] Improve the performance of the database select queries.

- Create indexes for animal_id columns to improve the speed of the first queries.
- Create indexes for animal_id, vet_id, and date_of_visit columns to improve the speed of the first and second queries
- Create indexes for email columns to improve the speed of the third queries

 ### Screenshots with EXPLAIN ANALYZE results (BEFORE)
Capture first
  ### Screenshots with EXPLAIN ANALYZE results (AFTER)
After 1
Capture second
## Missing screenshot (After)
After 2

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Raph, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is not contributed by all (3) assigned pair group members. Kindly ensure that all assigned members to this project contribute evenly to this task, and then request another code review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Davidosky007 — COMMENTED** [review]
Hi @RaphDasilva @danielmatama 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't show contribution from all team members this paired programming activity should have commits from three members but i can only see two, kindly engage all team members on this project  

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: data.sql [hunk 1]
```diff
@@ -112,5 +112,9 @@ INSERT INTO visits (animal_id, vets_id, visit_date)
             ((SELECT id FROM animals WHERE name = 'Blossom'),(SELECT id from vets WHERE name = 'Stephanie Mendez'), '2020-05-24'),
             ((SELECT id FROM animals WHERE name = 'Blossom'),(SELECT id from vets WHERE name = 'William Tatcher'), '2021-01-11');
 
+INSERT INTO visits (animal_id, vets_id, visit_date) SELECT * FROM (SELECT id FROM animals) animal_id, (SELECT id FROM vets) vets_id, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
```
Comment (Davidosky007) [inline data.sql:116]: * Kindly add an after screenshot for this query `SELECT * FROM visits where vet_id = 2;` as its not included in the PR

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @RaphDasilva @Fene-87 @danielmatama 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Well written PR 👍 
* Proper use of Gitflow 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @RaphDasilva, @danielmatama @Fene-87 👏 👏 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
Used `EXPLAIN ANALYZE` command for each query ✔️ 
 Added a data query screenshot to the PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**